konfios/konfswift/ui/BaseView.swift

```diff
@@ -10,34 +10,54 @@ import Foundation
 import konfios
 import UIKit
 
+var firstUpdateError = true
+
 extension UIViewController: KTBaseView {
     
     public func showError(error: KTStdlibThrowable) {
-        var message = ""
-        
+        var errorMessage: String? = nil
+        var popupMessage: String? = nil
+
         switch error {
         case is KTUnauthorized:
-            message = "Unauthorized"
+            errorMessage = "Unauthorized"
         case is KTCannotFavorite:
-            message = "Cannot set favorite now"
+            errorMessage = "Cannot set favorite now"
         case is KTCannotPostVote:
-            message = "Failed to post vote to server, please check your internet connection"
+            errorMessage = "Failed to post vote to server, please check your internet connection"
```

Comment:
I'd switch to using "rate/rating"

So 

Failed to rate sessions. Please check your connection